### ISSUE TITLE
Video date?

### ISSUE THREAD

**Issue by brunohenriquesk1** [LucasLeandro1204/Pornsearch#24]
is there any way to add when the video was posted? or sort it by the date? 

Thanks

**Comment by LucasLeandro1204** [comment]
@brunohenriquesk1 Yeah, I think it's possible in the most websites. I will add a date property to the search return data.

But I will not add any type of sorting by now. I really want to keep this version as simple as possible.

I'm thinking in a new l more complete version, with sorting, tags, etc.
